resources/pg_user.rb

```diff
@@ -13,3 +13,15 @@
 attribute :superuser,
 :kind_of => [TrueClass, FalseClass],
 :default => false
+
+attribute :admin_username,
```

Comment:
In pg_user we need to separate the user for chef connecting to the database in order to admin the server and the user being created to run the applications. I changed it in database.